include/matx/core/tensor.h

```diff
@@ -73,7 +73,7 @@ enum {
 #endif
 
 /**
- * @brief Iterator for TensorImpl for libraries that can take iterators as input (CUB).
+ * @brief Iterator around operators for libraries that can take iterators as input (CUB).
```

Comment:
should we move RandomOperatorIterator to its own source file?  It doesn't seem like it belongs in tensor.h

Reply:
Good idea. I was also going to combine the two iterators since we don't need two. One that has both l/rvalue operator() defined would suffice